webapi/src/main/scala/org/knora/webapi/slice/admin/domain/service/GroupService.scala

```diff
@@ -5,12 +5,62 @@
 
 package org.knora.webapi.slice.admin.domain.service
 
-import zio.ZLayer
+import zio._
+
+import dsp.errors.InconsistentRepositoryDataException
+import dsp.errors.NotFoundException
```

Comment:
**issue**: This exception should not be used in the domain package.

`NotFoundException` should only be used in the api package.

Reply:
Solved in [418e464](https://github.com/dasch-swiss/dsp-api/pull/3139/commits/418e464ddc8fd6ea4f9e1ca48ae5c392467766b3).